### PR TITLE
view_comment_ajax

### PR DESCRIPTION
**Pull request by datToc004**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by datToc004** [comment]
ready

File: resources/views/frontend/home/detail_dish.blade.php [hunk 1]
```diff
@@ -55,55 +57,72 @@ class="form-control input-number" value="1" min="1" max="100">
...
+                        <div id="reader">
+                            <ol>
+                                <li class="comments">
+                                    @foreach ($comments as $comment)
+                                        <div class="comment_box">
+                                            <a> <img src="/bower_components/start-bower123/images/avatar.jpg" alt="
+                                                avatar"> </a>
+                                            <div class="inside_comment">
+                                                <div class="comment-meta">
+                                                    <div class="commentsuser">{{ $comment->user->name }}</div>
```
Comment (vuminhhieu1311) [inline resources/views/frontend/home/detail_dish.blade.php:75]: Trong detailDish() ở controller chưa eager loading user

File: resources/views/frontend/home/detail_dish.blade.php [hunk 2]
```diff
@@ -55,55 +57,72 @@ class="form-control input-number" value="1" min="1" max="100">
...
+                                                    <div class="comment_date">{{ $comment->created_at }}</div>
+                                                </div>
+                                            </div>
+                                            <div class="comment-body">
+                                                <p>{{ $comment->note }}</p>
+                                            </div>
+                                            <div class="reply"> <a>Reply</a> </div>
```
Comment (vuminhhieu1311) [inline resources/views/frontend/home/detail_dish.blade.php:82]: i18n

File: resources/views/frontend/home/detail_dish.blade.php [hunk 3]
```diff
@@ -55,55 +57,72 @@ class="form-control input-number" value="1" min="1" max="100">
...
                         </div>
+                        <div id="respond">
+                            <form id="commentForm" method="post" class="cmxform">
+                                @csrf
+                                <div class="commentfields">
+                                    <h3>{{ __('messages.comment') }} <span>*</span></h3>
+                                    <textarea id="ccomment" class="comment-textarea required" name="note"></textarea>
+                                    {!! showError($errors, 'note') !!}
+                                    <input type="hidden" name="dish_id" value="{{ $dish->id }}" />
+                                    @if (isset(Auth::user()->id))
```
Comment (vuminhhieu1311) [inline resources/views/frontend/home/detail_dish.blade.php:103]:
```suggestion
                                    @if (isset(Auth::id()))
```

File: resources/views/frontend/home/detail_dish.blade.php [hunk 4]
```diff
@@ -55,55 +57,72 @@ class="form-control input-number" value="1" min="1" max="100">
...
+                                        <input type="hidden" name="user_id" value="{{ Auth::user()->id }}" />
+                                    @endif
+                                </div>
+                                <div class="commentfields">
+                                    <input class="commentbtn" type="submit" value="Post Comment">
```
Comment (vuminhhieu1311) [inline resources/views/frontend/home/detail_dish.blade.php:108]: i18n

**Pull request merged**